.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/e2eTest/resources/nexus-publish-e2e-minimal"]
+	path = src/e2eTest/resources/nexus-publish-e2e-minimal
+	url = git@github.com:gradle-nexus-e2e/nexus-publish-e2e-minimal.git
```

Comment:
What are the benefits of having this in a separate repo in a separate organization?

Reply:
A valid question. I initially created it just in the subpackage of `io.github.gradle-nexus` - `io.github.gradle-nexus.e2e`. However, doing the Nexus stuff I realized that it causes the staging profiles overlapping. As a result, using the production staging profile for `io.github.gradle-nexus` would allow releasing also e2e project (it applies also to all subpackages). However, in Nexus the release repository is bound to the staging profile (the second column from right).

![image](https://user-images.githubusercontent.com/148013/82140834-ec177080-9831-11ea-8f20-dd0d0266fccf.png)

As a result, it would be possible to accidentally release the e2e project to Maven Central, using my or your production credentials. Having a separate group - `io.github.gradle-nexus-e2e` (vs `io.github.gradle-nexus`) prevents that (the "No Sync Releases repo is always used).

I needed a separate organization to cover a domain requirement in Sonatype rules. However, after creating that PR, I've got a flash of inspiration that it is not a problem to move that e2e project back to the main organization (keeping just the empty organization in place for the domain requirement). I will update that, keeping the old `groupId` for the aforementioned reasons.